pkg/devfile/adapters/kubernetes/component/adapter.go

```diff
@@ -246,6 +248,22 @@ func (a Adapter) pushLocal(path string, files []string, delFiles []string, isFor
 	s := log.Spinner("Syncing files to the component")
 	defer s.End(false)
 
+	// If there's only one project defined in the devfile, sync to `/projects/project-name`, otherwise sync to /projects
+	projects := a.Devfile.Data.GetProjects()
+	fmt.Println(len(projects))
```

Comment:
Looks like an errant println slipped through

Reply:
That was an errant debugging line that slipped through, removed.